desktop/api/src/main/java/org/datacleaner/widgets/table/DCTable.java

```diff
@@ -307,11 +307,17 @@ public String getTextValueAt(int row, int column) {
 
     @Override
     public Object getValueAt(int row, int column) {
-        Object value = super.getValueAt(row, column);
-        if (value == null) {
-            value = LabelUtils.NULL_LABEL;
+        try {
+            Object value = super.getValueAt(row, column);
```

Comment:
Oh that try block shouldn't have been committed ... It was an experiment that I obviously forgot to clean up!